be/src/exprs/hash_functions.h

```diff
@@ -58,4 +64,33 @@ inline StatusOr<ColumnPtr> HashFunctions::murmur_hash3_32(FunctionContext* conte
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+inline StatusOr<ColumnPtr> HashFunctions::xx_hash3_64(FunctionContext* context, const starrocks::Columns& columns) {
+    std::vector<ColumnViewer<TYPE_VARCHAR>> viewers;
+
+    viewers.reserve(columns.size());
+    for (const auto& column : columns) {
+        viewers.emplace_back(column);
+    }
+
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_BIGINT> builder(size);
+    for (int row = 0; row < size; ++row) {
```

Comment:
suggest to transform this row-priority two-layer loops into column-priority loops, compiler shall try best to auto vectorize column-priority implement.

Reply:
nice catch ! I will do it soon